mozilla/fx-private-relay
# Start critical emails feature mpp 1347

This PR fixes https://mozilla-hub.atlassian.net/browse/MPP-1642, https://mozilla-hub.atlassian.net/browse/MPP-1641, https://mozilla-hub.atlassian.net/browse/MPP-1600.

## How to test:
The only way to test this end-to-end on your local Relay setup is to go all-out and get an email domain you control to go thru AWS SNS and something like ngrok to reach your local machine. (I can help walk thru this setup if anyone is interested)

So, we'll likely need to test this end-to-end on the dev server. But I’m also finding it hard to find a test list server that is 1. easy to sign up for testing 2. uses the List-Unsubscribe header on its first email, so I can verify an alias blocking list emails does NOT get it.
